mcp_server/__init__.py

```
@@ -1 +1 @@
-__all__ = [] 
\ No newline at end of file
+__all__ = []
```

Pylint (3.3.7)

[convention] Missing module docstring

(C0114)